popup.php

```diff
@@ -16,63 +16,61 @@
 
 global $wpdb;
 
-$table_name = $wpdb->prefix . "posts";
+$table_name = $wpdb->prefix . 'posts';
 
-$sql = "SELECT guid, post_mime_type FROM $table_name WHERE ID = " . (int) $_GET["attachment_id"];
+$attachment_id = absint( $_GET['attachment_id'] );
 
-list( $current_filename, $current_filetype ) = $wpdb->get_row( $sql, ARRAY_N );
+$sql = $wpdb->prepare( "SELECT guid, post_mime_type FROM $table_name WHERE ID = %d", $attachment_id );
 
-$current_filename = substr( $current_filename, ( strrpos( $current_filename, "/" ) + 1 ) );
+list( $current_filename, $current_filetype ) = $wpdb->get_row( $sql, ARRAY_N );
 
+$current_filename = substr( $current_filename, ( strrpos( $current_filename, '/' ) + 1 ) );
 
 ?>
 <div class="wrap">
 	<div id="icon-upload" class="icon32"><br/></div>
-	<h2><?php echo __( "Replace Media Upload", "enable-media-replace" ); ?></h2>
+	<h2><?php echo esc_html__( 'Replace Media Upload', 'enable-media-replace' ); ?></h2>
 
 	<?php
-	$url     = admin_url( "upload.php?page=enable-media-replace/enable-media-replace.php&noheader=true&action=media_replace_upload&attachment_id=" . (int) $_GET["attachment_id"] );
-	$action  = "media_replace_upload";
-	$formurl = wp_nonce_url( $url, $action );
+	$form_url = admin_url( 'upload.php?page=enable-media-replace/enable-media-replace.php&noheader=true&action=media_replace_upload&attachment_id=' . $attachment_id );
+
 	if ( FORCE_SSL_ADMIN ) {
-		$formurl = str_replace( "http:", "https:", $formurl );
+		$formurl = str_replace( 'http:', 'https:', $form_url );
```

Comment:
Worth keeping the var name further down too? Doesn't have to go in this batch of work